woocommerce/woocommerce-android
# Creating shipping label: Purchase section

Part of: #12281 

### Description
This PR includes the shipment label purchase section. Its changes are focused on UI, and the data used in the screens is mocked. I will connect this UI with real data in a different PR.

### Testing information

1. Create an Order with the processing status on a store that can create shipping labels
2. Tap on Orders
3. Tap on the order created on step 1
4. Tap con create shipping labels
5. Check that Shipment details card is displayed at the bottom of the screen
6. Tap on/Drag up Shipment details
7. Check that the bottom sheets expand and the shipment label purchase section is visible

### The tests that have been performed
1. Test that the UI section is visible

### Images/gif

| Before  | After |
| ------------- | ------------- |
| <img width="300" src="https://github.com/user-attachments/assets/d7489055-cbe7-4c35-84df-ba5a1b03e0f5" /> | <img width="300" src="https://github.com/user-attachments/assets/42680b63-9cc7-496f-a15a-79e00ced20bc" />  |



## Reviewer (or Author, in the case of optional code reviews):

Please make sure these conditions are met before approving the PR, or request changes if the PR needs improvement:
